src/librustc_driver/lib.rs

```diff
@@ -980,7 +980,7 @@ Available lint options:
     println!("Lint groups provided by rustc:\n");
     println!("    {}  {}", padded("name"), "sub-lints");
     println!("    {}  {}", padded("----"), "---------");
-    println!("    {}  {}", padded("warnings"), "all built-in lints");
+    println!("    {}  {}", padded("warnings"), "all lints activated to the warning level");
```

Comment:
Hmm, I don't find this phrasing very clear. "The warning level" sounds to me like something that one can control independently. Perhaps "all lints set to `warn`", or "all lints that are set to issue warnings"?

Reply:
If you have a lint that defaults to allow, but you set it explicitly to warn, and then you do `#[deny(warnings)]`, is that lint then denied?

Reply:
> Hmm, I don't find this phrasing very clear.

Yea, I struggled to find a good wording. Thanks for the suggestions, I used the second one for now.

> If you have a lint that defaults to allow, but you set it explicitly to warn, and then you do `#[deny(warnings)]`, is that lint then denied?

Yes.
```
#![deny(warnings)]
fn main() {
    Box::new(());
}
```
```
$ rustc -W box-pointers a.rs
error: type uses owned (Box type) pointers: std::boxed::Box<()>
 --> box.rs:3:5
  |
3 |     Box::new(());
  |     ^^^^^^^^^^^^
  |
note: lint level defined here
 --> box.rs:1:9
  |
1 | #![deny(warnings)]
  |         ^^^^^^^^
  = note: #[deny(box_pointers)] implied by #[deny(warnings)]

error: aborting due to previous error
```